examples/howto/plot_connectivity.py

```diff
@@ -71,7 +71,7 @@
 # directly.
 def get_network(probability=1.0):
     net = jones_2009_model(add_drives_from_params=True)
-    net.clear_connectivity()
+    net.connectivity = list()
```

Comment:
undo this since you have `clear_connectivity` function

Reply:
Sure will do

Reply:
Here net.connectivity needs to be completely cleared.
So both clear_connectivity() and clear_drives() will need to be called one by one.
- clear_connectivity() will leave connections of external drives open. For deleting them clear_drives will need to be called.